Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-current-contest.tsx

```diff
@@ -217,6 +217,11 @@ const CurrentContestsProvider = ({ children }: ICurrentContestsProviderProps) =>
             return;
         }
 
+        const { id } = contestToStart;
+        if (Number.isNaN(id)) {
```

Comment:
Is this needed? Is isNil not going to do the job here?

Reply:
Id is NaN here and isNil check does not cover it.